### PR TITLE
Fix focus on rename UI opening

### LINKED ISSUE #60035: New Rename UI doesn't auto-focus
When invoking the new preview rename UI, the textbox is not selected automatically.

### PR DESCRIPTION
**Pull request by ryzngard**
Previously the new adornment would not focus correctly. This changes to use the Load event and calls focus on the control directly.

It also cleans up some wording and usage of "Dashboard", which is not always correct now. 

Fixes #60035

### CODE REVIEW

File: src/EditorFeatures/Core.Wpf/InlineRename/CommandHandlers/RenameCommandHandler.cs [hunk 1]
```diff
@@ -13,6 +13,8 @@
 using Microsoft.CodeAnalysis.ErrorReporting;
 using Microsoft.CodeAnalysis.Extensions;
 using Microsoft.CodeAnalysis.Telemetry;
+using Microsoft.VisualStudio.Shell.Interop;
```
Comment (JoeRobich) [inline src/EditorFeatures/Core.Wpf/InlineRename/CommandHandlers/RenameCommandHandler.cs:16]: Is this necessary?

```suggestion
```

Comment (ryzngard) [reply src/EditorFeatures/Core.Wpf/InlineRename/CommandHandlers/RenameCommandHandler.cs]: Woops. Nope

**Review by JoeRobich — APPROVED** [review]

File: src/EditorFeatures/Core.Wpf/InlineRename/CommandHandlers/RenameCommandHandler.cs [hunk 2]
```diff
@@ -41,47 +42,49 @@ public RenameCommandHandler(IThreadingContext threadingContext, InlineRenameServ
...
             }
         }
 
-        protected override void SetDashboardFocusToPreviousElement(ITextView textView)
+        protected override void SetAdornmentFocusToPreviousElement(ITextView textView)
         {
-            if (GetDashboard(textView) is { } dashboard)
+            if (GetAdornment(textView) is RenameDashboard dashboard)
             {
                 dashboard.FocusNextElement();
```
Comment (JoeRobich) [inline src/EditorFeatures/Core.Wpf/InlineRename/CommandHandlers/RenameCommandHandler.cs:75]: Should this be `.FocusPreviousElement()`

Comment (ryzngard) [reply src/EditorFeatures/Core.Wpf/InlineRename/CommandHandlers/RenameCommandHandler.cs:75]: oh it definitely should... idk what that will break though :) I'll file an issue to change it

**Pull request merged**